app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue

```diff
@@ -0,0 +1,232 @@
+            v-model="values"
+            :placeholder="
+              $t('HELP_CENTER.ARTICLE_SETTINGS.FORM.META_TAGS.PLACEHOLDER')
+            "
+            label="name"
+            track-by="name"
+            :options="options"
+            :multiple="true"
+            :taggable="true"
+            @tag="addTagValue"
+          />
+        </label>
+      </div>
+      <div class="action-buttons">
+        <woot-button
+          icon="archive"
+          size="small"
+          variant="clear"
+          color-scheme="secondary"
+          @click="onClickArchiveArticle"
+        >
+          {{ $t('HELP_CENTER.ARTICLE_SETTINGS.BUTTONS.ARCHIVE') }}
+        </woot-button>
+        <woot-button
+          icon="delete"
+          size="small"
+          variant="clear"
+          color-scheme="alert"
+          @click="onClickDeleteArticle"
+        >
+          {{ $t('HELP_CENTER.ARTICLE_SETTINGS.BUTTONS.DELETE') }}
+        </woot-button>
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import MultiselectDropdown from 'shared/components/ui/MultiselectDropdown';
+export default {
+  components: {
+    MultiselectDropdown,
+  },
+  data() {
+    return {
+      // Dummy value
+      categoryList: [
+        {
+          id: 1,
+          name: 'Getting started',
+        },
+        {
+          id: 2,
+          name: 'Features',
+        },
+      ],
+      selectedCategory: {
+        id: 1,
+        name: 'Features',
+      },
+      authorList: [
+        {
+          id: 1,
+          name: 'John Doe',
+        },
+        {
+          id: 2,
+          name: 'Jane Doe',
+        },
+      ],
+      assignedAuthor: {
+        id: 1,
+        name: 'John Doe',
+      },
+      title: '',
+      description: '',
+      values: [],
+      options: [],
+    };
+  },
+  methods: {
+    addTagValue(tagValue) {
+      const tag = {
+        name: tagValue,
+      };
+      this.values.push(tag);
+      this.$refs.tagInput.$el.focus();
+    },
+    onClickSelectCategory() {
+      this.$emit('select-category');
+    },
+    onClickAssignAuthor() {
+      this.$emit('assign-author');
+    },
+    onClickArchiveArticle() {
+      this.$emit('archive-article');
+    },
+    onClickDeleteArticle() {
+      this.$emit('delete-article');
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.article-settings--container {
+  flex: 0.3;
+  min-width: var(--space-giga);
+  overflow: scroll;
+  border-left: 1px solid var(--color-border-light);
+  margin-left: var(--space-normal);
+  padding-left: var(--space-normal);
+  padding-top: var(--space-small);
+  padding-bottom: var(--space-small);
+
+  .form-wrap {
+    margin-top: var(--space-normal);
+    margin-bottom: var(--space-medium);
+
+    textarea {
+      font-size: var(--font-size-small);
+    }
+  }
+
+  .action-buttons {
+    display: flex;
+    flex-direction: column;
+
+    ::v-deep .button {
+      padding-left: 0;
+    }
+  }
+}
+::v-deep {
+  .multiselect {
```

Comment:
Can you check if  `multiselect-wrap--small` or `multiselect-wrap--medium` wrapper class to avoid them? 
Used here - `app/javascript/dashboard/components/widgets/AutomationActionInput.vue`

Reply:
@nithindavid 
Tried it looks like this

**Medium**

<img width="632" alt="Screenshot 2022-08-08 at 3 19 31 PM" src="https://user-images.githubusercontent.com/64252451/183390833-5b0a4e2e-cd80-4ff2-b4fe-17ef0ef74fbd.png">

**Small**

<img width="632" alt="Screenshot 2022-08-08 at 3 20 46 PM" src="https://user-images.githubusercontent.com/64252451/183390882-bb9ba361-50e8-410e-91ee-3f454446e28a.png">


I tried removing that icon from the right side and that popover when we type something.

Reply:
Okay in this case can we use the class and override necessary css in this component ?

Reply:
Tried it styles of multi-select component is breaking.

Reply:
You can see it here

https://user-images.githubusercontent.com/64252451/183698043-125637c1-a044-48cc-880c-83381e4a6a7e.mov